src/main.rs

```diff
@@ -1,3 +1,6 @@
+use std::str::FromStr;
```

Comment:
Could you run the remove the unused import please :)

Reply:
opps! will fix shortly